Detections/MultipleDataSources/ADFS-DKM-MasterKey-Export.yaml

```diff
@@ -1,9 +1,43 @@
 id: 18e6a87e-9d06-4a4e-8b59-3469cd49552d
 name: ADFS DKM Master Key Export
 description: | 
-  'Identifies an export of the ADFS DKM Master Key from Active Directory.
-   References: https://blogs.microsoft.com/on-the-issues/2020/12/13/customers-protect-nation-state-cyberattacks/, 
-   https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html?1'
+  Identifies an export of the ADFS DKM Master Key from Active Directory.
+  References: https://blogs.microsoft.com/on-the-issues/2020/12/13/customers-protect-nation-state-cyberattacks/, 
+  https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html?1
+
+  # Windows Security Event 4662 Requirement:
```

Comment:
This has become very long for a description and may not fit in the UX.  Do you think we could put the link to the Conversation tab of the original PR and this PR for folks to read instead of including here?

Reply:
Yes that works. let me update the original (first) PR which has all the background behind this query and then just put the PR link in the description section.